index.html

```diff
@@ -2,160 +2,217 @@
+                            alt="" srcset=""></a></li>
+                <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (4).svg"
+                            alt="" srcset=""></a></li>
+                <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (2).svg"
+                            alt="" srcset=""></a></li>
+            </ul>
+            <div class="see-resume">
+                <button><a href="#">See resume</a></button>
+            </div>
+            <div class="languages">
+                <div class="flex">
+                    <p>Languages</p>
+                    <i class="fa-solid fa-angle-down "></i>
+                </div>
+                <ul class="mycompetence flex">
+                    <li class="flex"><img src="/assest/img/Ellipse 1.png" alt=""> JavaScript</li>
+                    <li class="flex"> <img src="/assest/img/Ellipse 1 (1).png" alt="">HTML</li>
+                    <li class="flex"> <img src="/assest/img/Ellipse 1 (2).png" alt="">CSS</li>
+                </ul>
+                <div class="flex border-bootom">
+                    <p>Framwrks</p>
+                    <i class="fa-solid fa-angle-down "></i>
+                </div>
+                <div class="flex border-bootom">
+                    <p>Skills</p>
+                    <i class="fa-solid fa-angle-down "></i>
+                </div>
+            </div>
+        </div>
+        </div>
+    </section>
+
+    <section class="contact-form">
+        <div class="contac-card">
+            <div class="contact-me">
+                <h2>contact me</h2>
+
+                <p>
+                    if you have an application you are interested in developing, a
+                    features that you need built or a project that need coding. I'd love
+                    to help with it.
+                </p>
+            </div>
+            <div class="contact-me-form">
+                <form>
+                    <div class="username-input">
+                        <input type="text" name="username" class="input" placeholder="enter username" maxlength="30"
+                            required />
+                    </div>
+                    <br />
+                    <div class="email-input">
+                        <input type="email" name="email" class="input" placeholder="enter email"
+                            pattern="^[a-zA-Z0-9]+@domain\.[a-zA-z]+$" size="64" maxlength="64" required />
+                    </div>
+                    <br />
+                    <div class="text-area">
+                        <textarea name="message" type="text" class="input-textarea" cols="30" maxlength="500"
+                            placeholder="enter your message here" required></textarea>
+                    </div>
+                    <button type="button" value="Get In Touch" class="submit-button">
```

Comment:
- [ ] In order to submit your form successfully ✔, make sure you set the type of your button to `submit` or leave it with no type attribute, as the default type is set to **submit** when no type attribute is specified 😊.